### ISSUE TITLE
Argument expansion for home directory does not work.

### ISSUE THREAD

**Issue by mriesch-wv** [pengutronix/umpf#13]
I tried to generate patches with `umpf format-patch --patchdir=~/path/to/BSP my_tag` but instead of expanding `~` to my home directory, a directory called `~` was created in my current directory.

**Comment by michaelolbrich** [comment]
Hmmm, the problem is, that in umpf I cannot tell the difference between:
```
umpf format-patch --patchdir=~/path/to/BSP my_tag
```
and
```
umpf format-patch --patchdir "~/path/to/BSP my_tag"
```
The first one is your use case where expansion is expected. In the second case, the expansion is explicitly prevented.
I suggest you use `... --patchdir ~/path/to/BSP ...` instead. This way the shell will expand `~` for you.

While looking at that, I noticed, that the bash completion helper never made it into the public repo. I've created #15 to change that. And I modified the help text in a way that it will complete with a space instead of '=' so it will work like you would expect it.
